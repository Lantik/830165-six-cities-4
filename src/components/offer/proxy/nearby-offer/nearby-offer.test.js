import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import NearbyOffer from './nearby-offer.jsx';
import Offer from '../../offer.jsx';
import {doWithoutPropTypes} from '../../../../utils/testing/testing.js';

const mock = {
  offer: {}
};

it(`<NearbyOffer/> rendered correctly`, () => doWithoutPropTypes([Offer], () => {
  const {offer} = mock;
  const renderer = new ShallowRenderer();

  renderer.render(<NearbyOffer
    offer={offer}
    onMouseEnter={() => {}}
    onOfferHeaderClick={() => {}}
    articleClassName={`test-class`}
    imageWrapperClassName={`test-class`}
  />);
  const tree = renderer.getRenderOutput();

  expect(tree).toMatchSnapshot();
}));
