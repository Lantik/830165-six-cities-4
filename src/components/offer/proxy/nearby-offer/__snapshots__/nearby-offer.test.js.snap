// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NearbyOffer/> rendered correctly 1`] = `
<Offer
  articleClassName="test-class"
  imageWrapperClassName="test-class"
  offer={Object {}}
  onMouseEnter={[Function]}
  onOfferHeaderClick={[Function]}
/>
`;
